### PR TITLE
feat(chore): add project tools and utilities

### PR DESCRIPTION
**Pull request by ev1stensberg**
**What kind of change does this PR introduce?**

Feature
**Did you add tests for your changes?**
n/a
**If relevant, did you update the documentation?**
n/a
**Summary**
Adds project utilities like commitlint, changelog 

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by ev1stensberg** [comment]
@paulirish is it okay with you guys if we use your linting + changelog implementation? Credited you guys at every place we use code from lighthouse

File: package.json [hunk 1]
```diff
@@ -29,11 +29,15 @@
     "appveyor:lint": "yarn lint",
     "appveyor:test": "nyc jest && yarn reportCoverage",
     "travis:integration": "yarn prepare && nyc jest && yarn reportCoverage",
-    "travis:lint": "yarn lint"
+    "travis:lint": "yarn lint && yarn bundlesize",
+    "bundlesize": "bundlesize",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "changelog": "conventional-changelog --config ./build/changelog-generator/index.js --infile changelog.md --same-file"
```
Comment (ematipico) [inline package.json]: `changelog.md` => `CHANGELOG.md`

File: package.json [hunk 2]
```diff
@@ -29,11 +29,15 @@
     "appveyor:lint": "yarn lint",
     "appveyor:test": "nyc jest && yarn reportCoverage",
     "travis:integration": "yarn prepare && nyc jest && yarn reportCoverage",
-    "travis:lint": "yarn lint"
+    "travis:lint": "yarn lint && yarn bundlesize",
+    "bundlesize": "bundlesize",
```
[the remark this answers is not in the document]
Comment (fokusferit) [reply package.json]: another question: are we adding bundlesize so users can use it?

[the remark this answers is not in the document]
Comment (ev1stensberg) [reply package.json]: It's for CI's for now, we could add it to as a feature to webpack, but that needs some work

**Pull request merged**